decred_wallet/view_util/customUImage.swift

```diff
@@ -1,75 +1,105 @@
 //
-//  iOSDevCenters+GIF.swift
-//  GIF-Swift
-//
-//  Created by iOSDevCenters on 11/12/15.
-//  Copyright © 2016 iOSDevCenters. All rights reserved.
-//
+//  customUImage.swift
+//  Copyright (c) 2014 Arne Bahlo. All rights reserved.
```

Comment:
What is this copyright about? Decred copyright header should also be added to this file since we have contributed to it.